utils/sentry_client.py

```diff
@@ -0,0 +1,155 @@
+import requests
+import json
+
+class SentryClient:
+  ORGANIZATION = "sentry"
+
+  def __init__(self, host, token):
+    self.host = host
+    self.auth_token = token
+
+  def do_sentry_api_call(self, method, path, slugs, payload=None):
+    success_codes = {"get": 200, "post": 201, "delete": 204, "put": 200}
+    url = "%s/api/0/%s/" % (self.host, path)
+
+    if len(slugs) > 0:
+      url = "%s%s/" % (url, "/".join(slugs))
+
+    headers = {"Authorization": "Bearer " + self.auth_token}
+    call = getattr(requests, method, None)
+    if call == None:
+      print ("invalid http method %s" % method)
```

Comment:
perhaps we should raise an exception here? In any case we try to never do `print`, I would suggest `logging.error` in this case, but I personally prefer raising an exception.

Reply:
Agreed, exception is really the path to take here